src/app/App.tsx

```diff
@@ -3,9 +3,13 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import ShoppingList from './pages/ShoppingList/ShoppingList';
 import Inventory from './pages/Inventory/Inventory';
 import RecipeDetail from './pages/RecipeDetail/RecipeDetail';
+import RecipeForm from './components/RecipeForm/RecipeForm';
 
 export default (): JSX.Element => (
   <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<RecipeForm />} />
+    </Routes>
     <Routes>
       <Route path="/shopping-list" element={<ShoppingList />} />
     </Routes>
```

Comment:
You only need one `Routes` and then you have several `Route`'s ;) 

```suggestion
    <Routes>
      <Route path="/" element={<RecipeForm />} />
      <Route path="/shopping-list" element={<ShoppingList />} />
```
So the whole thing would look like this: 

```
<BrowserRouter>
    <Routes>
      <Route path="/" element={<RecipeForm />} />
      <Route path="/shopping-list" element={<ShoppingList />} />
      <Route path="/inventory" element={<Inventory />} />
      <Route path="/1" element={<RecipeDetail />} />
    </Routes>
  </BrowserRouter>
```

Reply:
hahaha thanks, I really thought it has to be this way 😬😬😅
I'll change that :)